ros/src/computing/perception/detection/lib/fusion/fusion.cpp

```diff
@@ -311,13 +311,31 @@ void setPointsImage(const points2image::PointsImage& points_image)
 #endif
 	points_msg = points_image;//store vscan pointcloud
 	pointsStored = false;
+
+	/*
+	 * Reset 2D vector
+	 */
+	for (auto i=0; i<g_scan_image.distance.size(); i++) {
```

Comment:
This line causes warning. Type of `i` is `int` by type conference because `0` is `int`. While type of `g_scan_image.distance.size()` is `size_t`(unsigned  type). There is a comparison between signed and unsigned type. You should use `size_t` for `i` type. Or use `ranged-for`.